packages/expo-updates/package.json

```diff
@@ -12,6 +12,8 @@
     "test": "expo-module test",
     "prepare": "expo-module prepare",
     "prepublishOnly": "expo-module prepublishOnly",
+    "prepack": "expo-module vendorScripts",
```

Comment:
The expo-module-scripts binaries are named after npm lifecycle scripts: `expo-module prepack`. The idea is that each package's package.json file shouldn't make many decisions -- they should just map each npm lifecycle script to `expo-module <X>`.